src/jit/ryujit_x86/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+project(ryujit)
+
+remove_definitions(-DFEATURE_MERGE_JIT_AND_ENGINE)
+
+add_library(ryujit_x86
```

Comment:
We will need the same flavor for ARM too. Shouldn't this rather be called something like `ryujit_newbackend` or `ryujit_unstable` so that the same name works for both x86 and arm (and it is obvious from the name what it actually contains)?